Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,16 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
+using firebase::firestore::model::DatabaseId;
 using firebase::firestore::remote::Serializer;
 
 namespace {
 
 // Fuzz-test the deserialization process in Firestore. The Serializer reads raw
 // bytes and converts them to a model object.
 void FuzzTestDeserialization(const uint8_t *data, size_t size) {
-  // TODO(minafarid): fuzz-test Serializer.
+  DatabaseId database_id{"project", DatabaseId::kDefault};
+  Serializer serializer(database_id);
```

Comment:
It's mostly irrelevant now, but to finish our offline discussion: if you wanted to reuse the same `serializer` across invocations, another option would be to have it as a `static` local variable:
```cpp
void FuzzTestDeserialization(const uint8_t *data, size_t size) {
  DatabaseId database_id{"project", DatabaseId::kDefault};
  static Serializer serializer(database_id);
```
It would be initialized once, upon the first call of the function, and reused on subsequent calls. The advantage would be avoiding global state.